manim/animation/animation.py

```diff
@@ -21,9 +21,30 @@
 
 
 class Animation:
+    def __new__(
+        cls,
+        mobject: Optional[Mobject] = None,
+        *args,
+        use_override: bool = True,
+        **kwargs,
+    ):
+        if isinstance(mobject, Mobject) and use_override:
```

Comment:
Nonlblocking but changing the check to
```suggestion
        if mobject is not None and use_override:
```
makes more sense to me

Reply:
mm, that makes sense